server/sonar-server/src/main/java/org/sonar/server/component/ws/SearchProjectsAction.java

```diff
@@ -325,6 +326,18 @@ private SearchProjectsWsResponse buildResponse(SearchProjectsRequest request, Se
     return wsResponse;
   }
 
+  private static void addMandatoryValuesToFacet(Facets facets, String facetName, @Nullable Iterable<String> mandatoryValues) {
```

Comment:
why a @Nullable Iterable? I'd expect either an empty Iterable or no presence of the Optional query.getLanguages()